rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -199,6 +201,24 @@ void getActiveFederatorPublicKeyOfType_returnsFederatorPublicKeysFromGenesisFede
             byte[] activeFederatorMstPublicKey = federationSupport.getActiveFederatorPublicKeyOfType(0, KeyType.MST);
             assertThat(activeFederatorMstPublicKey, is(federatorFromGenesisFederationMstPublicKey.getPubKey(true)));
         }
+
+        @Test
+        @Tag("getActiveFederationBtcUTXOs")
+        void getActiveFederationUTXOs_returnsGenesisFederationUTXOs() {
+            storageProvider = mock(FederationStorageProviderImpl.class);
```

Comment:
Any reason to mock instead of using the inMemoeryStorage as in the other test?

Reply:
yes! i added a disclaimer in the pr description:

Disclaimer:
The values being obtained from the provider had to be mocked because there are no setFederationUTXOs methods in the implementation

Reply:
Sorry, didn't see that 😬

What about setting the value directly in the storage under the proper key? Like this https://github.com/rsksmart/rskj/blob/federation-support-refactor-integration/rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java#L669-L673

Reply:
jaja no worries :))
i thought about doing that, but i wasn't convinced. But yes i think is more realistic doing it that way, ill change it. Thanks!

Reply:
done